packages/common/src/store/pages/saved-page/lineups/tracks/reducer.ts

```diff
@@ -1,12 +1,14 @@
 // @ts-nocheck
 // TODO(nkang) - convert to TS
+import { Status } from 'models'
 import { RESET_SUCCEEDED, stripPrefix } from 'store/lineup/actions'
 import { initialLineupState } from 'store/lineup/reducer'
 import { PREFIX } from 'store/pages/saved-page/lineups/tracks/actions'
 
 const initialState = {
   ...initialLineupState,
-  prefix: PREFIX
+  prefix: PREFIX,
+  status: Status.LOADING
```

Comment:
cc @dylanjeffers cause you just touched the lineup initial status. Is this the best way to do this?

Reply:
yeah this seems like a good idea

Reply:
Good call out. thank you both! 🙏